test/e2e/provisioning/all_e2e_test.go

```diff
@@ -152,3 +154,39 @@ func TestVsphereProvisioningE2E(t *testing.T) {
 	}
 	runScenarios(t, excludeSelector, params, vs_manifest, fmt.Sprintf("vs-%s", *testRunIdentifier))
 }
+
+// TestVsphereProvisioning - a test suite that exercises vsphere provider
+// by requesting nodes with different combination of container runtime type, container runtime version and the OS flavour.
+func TestVsphereStaticIPProvisioningE2E(t *testing.T) {
+	// test data
```

Comment:
Please add `t.Parallel()` here

Reply:
`testScenario` already adds `t.Parallel()`, and there are no subtest here, so we would be running `t.Parallel()` twice, causing a panic. That being said, I just noticed that the function's comment is wrong - I'll fix it promptly.